validation/README.md

```diff
@@ -0,0 +1,24 @@
+# CONSISTENCY VALIDATION GUIDE
+
+## STEP 1: Deploy a multiple-node working environment
+
+Example:
+- Create 4 Instances (1 us-east-1, 1 us-east-2, 1 us-west-1, 1 us-west-2)
+- Configure the number of sync and async nodes and latency threshold
+    - Change the parameters in `scripts/test_infra/create_test_instance.sh`
+    - Parameters: `localreplicanum`, `remotereplicanum`, `remotestorelatencythresholdinmillisec`
+- Modify read behaviors if necessary
```

Comment:
any example on the read behavior modification?

Reply:
an example was added